api/src/main/resources/workbench.yaml

```diff
@@ -2098,33 +2125,55 @@ definitions:
       sortColumn:
         type: string
 
-  ParticipantCohortStatusesRequest:
-    type: object
+  PageFilterRequest:
+    discriminator: pageFilterType
     required:
       - page
       - pageSize
-      - sortColumn
       - sortOrder
+      - pageFilterType # required for inheritance to work
     properties:
       page:
         type: integer
         description: specific page (default is 0)
       pageSize:
         type: integer
         description: page size of results (default is 25)
-      sortColumn:
-        $ref: "#/definitions/ParticipantCohortStatusColumns"
-        description: Column upon which to sort (default is 'participantId')
       sortOrder:
         $ref: "#/definitions/SortOrder"
-        description: Sort direction (default is ascending)
+      pageFilterType:
+        $ref: "#/definitions/PageFilterType"
       filters:
         $ref: "#/definitions/FilterList"
 
+  ParticipantCohortStatusesPageFilter:
+    allOf:
+      - $ref: '#/definitions/PageFilterRequest'
+      - properties: # extra properties
+          sortColumn:
+            $ref: "#/definitions/ParticipantCohortStatusColumns"
```

Comment:
We could just have the sort column be a string and not worry about having different objects for different page filters. WDYT?

Reply:
Chatted offline. For the materialize cohort endpoint, we're forced in this direction because we want one endpoint to handle all OMOP tables. If we have per-table endpoints we can do this... whether it's a better idea to do things this way or just have some documentation you can point @johnstill etc. at for the right column names, I dunno. :)